src/components/database/settings/driver-config/EditDriverConfigForm.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { Modal, Form, Input } from 'antd';
+import FormItemLabel from '../../../form-item-label/FormItemLabel';
+
+const EditDriverConfigForm = ({ handleSubmit, handleCancel, dbType, initialValue }) => {
+
+  const [form] = Form.useForm()
+
+  const handleSubmitClick = () => {
+    form.validateFields().then(values => {
+      handleSubmit(values)
+      handleCancel()
+    })
+  }
+
+  return(
+    <Modal
+    title='Configure database driver'
+    visible={true}
+    onOk={handleSubmitClick}
+    onCancel={handleCancel}
+    okText='Save'>
+      <Form form={form} initialValues={initialValue} onFinish={handleSubmitClick}>
+        <FormItemLabel name='Max connections (default: 100)' />
```

Comment:
The `(default: 100)` text is of lighter colour in the design. `FormItemLabel` has a prop called `hint` for this purpose. Provide these default values in the `hint`.